### ISSUE TITLE
Blend file corrupted when saving to shared storage

### ISSUE THREAD

**Issue by ajgeiss0702** [armadillica/flamenco#265]
Hello there

I have several workers set up, and they appear to function correctly (test renders work), yet if I try to send a render to them via blender cloud, it seems like the .blend file always gets corrupted. The original file is fine, yet if I try to open the file that is saved in the flamenco storage, i get the same error that the workers get.
```
Error: File format is not supported in file '/flamenco/jobs/2022-01-24-222534.034175-ajgeiss72-test.flamenco/test.flamenco.blend'
```

Any ideas what might cause this? And what possible fixes there are?

**Comment by RandKmiec** [comment]
Not sure if this is the same issue, but I was told there was a bug (present in at least 1.23 and fixed in 1.24) of the Blender Cloud plugin where it was compressing .blend files twice; so that the Blender file (/flamenco/jobs/2022-01-24-222534.034175-ajgeiss72-test.flamenco/test.flamenco.blend) would actually be a gzip'd version.  If you are able to open it up in 7zip and see a .blend file inside (which opens fine), this is likely your problem.  Updating to 1.24 and restarting Blender and the worker fixed it for me.

**Comment by sybrenstuvel** [comment]
Yup, that's been fixed in 1.24, thanks for the report!
